ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/offers/PartStatementMapper.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.ccd.mapper.offers;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.offers.CCDMadeBy;
+import uk.gov.hmcts.cmc.ccd.domain.offers.CCDPartyStatement;
+import uk.gov.hmcts.cmc.ccd.domain.offers.CCDStatementType;
+import uk.gov.hmcts.cmc.ccd.mapper.Mapper;
+import uk.gov.hmcts.cmc.domain.models.offers.MadeBy;
+import uk.gov.hmcts.cmc.domain.models.offers.PartyStatement;
+import uk.gov.hmcts.cmc.domain.models.offers.StatementType;
+
+@Component
+public class PartStatementMapper implements Mapper<CCDPartyStatement, PartyStatement> {
+
+    private final OfferMapper offerMapper;
+
+    public PartStatementMapper(OfferMapper offerMapper) {
+        this.offerMapper = offerMapper;
+    }
+
+    @Override
+    public CCDPartyStatement to(PartyStatement partyStatement) {
+        CCDPartyStatement.CCDPartyStatementBuilder builder = CCDPartyStatement.builder();
+        builder.type(CCDStatementType.valueOf(partyStatement.getType().name()));
+        builder.madeBy(CCDMadeBy.valueOf(partyStatement.getMadeBy().name()));
```

Comment:
Same as Tim's previous comment -- builder calls can be chained. Please cover it in all places where calls are not chained.